app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,27 @@
+class SessionsController < ApplicationController
+    def create
+        user = User.find_or_create_by(email: user_params[:email])
+        user.update(user_params)
+        session[:user_id] = user.id
+        redirect_to root_path
+    end
+
+    # def destroy
```

Comment:
Will we be able to implement `#destroy` during the MVP? If not, maybe remove in a future PR